### ISSUE TITLE
Investigate which TFM's we want to support in 2.0.0

### ISSUE THREAD

**Issue by akamsteeg** [akamsteeg/AtleX.CommandLineArguments#10]
Currently, the library supports three Target Framework Monikers:

1. NET45
2. NET46
3. NETSTANDARD1.5

However, this seems a bit redundant. If we support .NET 4.5, the lowest full framework TFM we can target at the moment, we also implicitly support 4.6, 4.6.1, 4.7, etc. Why do we explicitly specify .NET 4.6 then?

With .NET Standard 1.5 we can run on the full framework in any application [targeting .NET 4.6.2 or higher and in .NET Core 1.0 or higher](https://docs.microsoft.com/en-us/dotnet/standard/net-standard). For consumers being stuck (for the moment?) at older versions of .NET we can give them 4.5. 

So, in summary: Wouldn't it be enough to just target NET45 and NETSTANDARD1.5?

**Comment by akamsteeg** [comment]
I suggest we leave it at NET45, which we can run on NET46 or higher too, and NETSTANDARD1.5.
